index.js

```diff
@@ -737,9 +737,16 @@ module.exports = class Hypercore extends EventEmitter {
       end = start + 1
     }
 
-    if (start >= end) return
+    if (start >= end) return 0
+
+    let bytes = 0
+    if (opts && opts.diff) bytes = (await this.info({ storage: true })).blocks
 
     await this.core.clear(start, end)
+
+    if (opts && opts.diff) bytes -= (await this.info({ storage: true })).blocks
+
+    return bytes
```

Comment:
should return { blocks: bytes } so it can tell for each file in the future

Reply:
can also be done a lot faster, by exposing a static helper from Info to get the bytesUsed and just doing it with the blocks file